src/libs/Navigation/types.ts

```diff
@@ -460,20 +460,26 @@ type BottomTabNavigatorParamList = {
     [SCREENS.WORKSPACE.INITIAL]: undefined;
 };
 
-type PublicScreensParamList = {
+type SharedScreensParamList = {
     [NAVIGATORS.BOTTOM_TAB_NAVIGATOR]: NavigatorScreenParams<BottomTabNavigatorParamList>;
     [SCREENS.TRANSITION_BETWEEN_APPS]: {
-        email?: string;
+        email: string;
+        accountID: number;
         error?: string;
-        shortLivedAuthToken?: string;
-        shortLivedToken?: string;
+        supportAuthToken: string;
```

Comment:
looks like most of these should be optional. Keep in mind that these are basically URL params. We won't always have a `supportAuthToken`, so that should be optional:

```suggestion
        supportAuthToken?: string;
```